Makefile.js

```diff
@@ -622,7 +622,12 @@ target.gensite = function(prereleaseVersion) {
 
     // 3. Copy docs folder to a temporary directory
     echo("> Copying the docs folder (Step 3)");
-    cp("-rf", "docs/*", TEMP_DIR);
+    docFiles.forEach(filePath => {
+        cp("-rf", `docs/src${filePath}*`, TEMP_DIR);
+    });
```

Comment:
This will copy all files to the same directory?

`docs/src/user-guide/command-line-interface.md` -> `TEMP_DIR/command-line-interface.md`
`docs/src/rules/accessor-pairs.md` -> `TEMP_DIR/accessor-pairs.md`

Reply:
Ooh good catch. Didn't show up when I tried this locally due to previous files already being there.

> We didn't use a simpler `cp("-rf", "docs/src/*", TEMP_DIR);` because it wouldn't skip `pages/`?

Correct.